README.md

```diff
@@ -1,20 +1,222 @@
+
+       CREATE SCHEMA IF NOT EXISTS my_custom_schema;
+
+Within your application you will need to extend two base classes provided in the Housekeeping project: `LegacyReplicaPath` and `LegacyReplicaPathRepository`.
+These classes provide skeleton implementations of the LegacyReplicaPath `Entity` and the LegacyReplicaPathRepository `CrudRepository`.
+
+The `LegacyReplicaPath` should be extended when the user wishes to store their legacy paths in a database schema with a name other than `housekeeping`, with a table name other than `legacy_replica_path` or both. The class which extends LegacyReplicaPath must be annotated with the `@Entity` annotation and the `@Table` annotation. An example is given below which will provide the basis for creating a schema named `my_custom_schema` in your database, and a table named `legacy_replica_path` within the `my_custom_schema` schema.  
+
+      @Entity
+      @Table(schema = "my_custom_schema", name = "legacy_replica_path",
+          uniqueConstraints = @UniqueConstraint(columnNames = { "path", "creation_timestamp" }))
+      public class MyJobsLegacyReplicaPath extends EntityLegacyReplicaPath {
+
+        protected MyJobsLegacyReplicaPath() {}
+
+        public MyJobsLegacyReplicaPath(String path) {
+          this.path = path;
+          this.eventId = "";
+          this.pathEventId = "";
+        }
+      }
+
+To accompany the custom `LegacyReplicaPath` implementation the user should extend the `LegacyReplicaPathRepository` interface providing the custom `LegacyReplicaPath` implementation as a generic type argument. This simplifies the creation of a `CrudRepository` for your `LegacyReplicaPath`.
+For example:
+
+      public interface MyJobLegacyReplicaPathRepository
+          extends LegacyReplicaPathRepository<MyJobsLegacyReplicaPath> {
+      }
+
+Whether you are using a custom housekeeping configuration, or the defaults, your application must provide two crucial annotations which will load the Entities and CrudRepositories that you require. These are the `@EntityScan` and `@EnableJpaRepositories` annotations. These annotations are best demonstrated in an example:
+
+    //The class annotated with `@Entity` that defines the required LegacyReplicaPath implementation
+    @EntityScan(basePackageClasses = HousekeepingLegacyReplicaPath.class)
+    //The class which extends LegacyReplicaPathRepository and contains your desired `CrudRepository` implementation
+    @EnableJpaRepositories(basePackageClasses = HousekeepingLegacyReplicaPathRepository.class)
+
+
+## Password Encryption
+Housekeeping allows users to provide encrypted passwords in their configuration or programs. The Housekeeping project has the jasypt-1.9.2.jar in the /lib directory. This jar can be used to generate encrypted passwords which can be decrypted by Spring Boots jasypt support in your application. The necessary dependencies for decrypting jasypt encrypted passwords are packaged with the Housekeeping project.
```

Comment:
"The necessary dependencies for decrypting jasypt encrypted passwords are packaged with the Housekeeping project." If you make the above change you can probably remove this line.